lib/crypto.js

```diff
@@ -223,22 +223,4 @@ Object.defineProperties(exports, {
     enumerable: true,
     value: constants
   },
```

Comment:
The rest of the file does not use trailing commas, please remove this. I don't want to start any new discussions about trailing commas, but I'd go for consistency in this case.

Reply:
The line right above doesn't have a trailing comma either, but @bnoordhuis suggests we use trailing commas moving forwards?

Idk, sounds reasonable to just add them wherever you go.

Reply:
Are you referring to https://github.com/nodejs/node/pull/20703#issuecomment-388754052? ESLint still doesn't have a file-by-file rule as far as I know. Personally, I am not a fan of dangling commas, and we discussed that topic numerous times (e.g. https://github.com/nodejs/node/issues/19131#issuecomment-370429515). I totally understand Ben's reasoning, but this kind of "churn" isn't that common from my perspective, correct me if I am wrong.

Reply:
Yeah, let's not. We should probably skip the comma for consistency, you're right.